src/main/java/org/beanpod/switchboard/controller/ChannelController.java

```diff
@@ -0,0 +1,4 @@
+package org.beanpod.switchboard.controller;
+
+public class ChannelController {
```

Comment:
Why is this class empty?

Reply:
I removed it. We don't need it as of right now.

Reply:
Removed it. We dont need it for now.